### PR TITLE
Update docs about when emails for changed credentials are sent

### LINKED ISSUE #27620: Incomplete documentation when an email about changed credentials is sent
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

docs

### Describe the bug

The documentation doesn't mention that emails are only sent when an email is marked as verified for the user. 

This is a follow-up to #21817

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

Documentation is complete to describe the observed behavior

### Actual behavior

Documentation is incomplete

### How to Reproduce?

* Enable email for the realm
* Enable email event listener
* Change the user's password 

### Anything else?

I'll prepare a PR

### PR DESCRIPTION
**Pull request by ahus1**
Closes #27620

### CODE REVIEW

**Review by jonkoops — DISMISSED** [review]

**Review by mposolda — DISMISSED** [review]

**Comment by ahus1** [comment]
@andymunro - could you please review the text changes when you have the time? Thanks!

File: docs/documentation/server_admin/topics/events/login.adoc [hunk 1]
```diff
@@ -159,12 +159,21 @@ The valid values for log levels are `debug`, `info`, `warn`, `error`, and `fatal
 
 ===== The Email Event Listener
 
-The Email Event Listener sends an email to the user's account when an event occurs and supports the following events:
+The Email Event Listener sends an email message to the user's email address when an event occurs and supports the following events:
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/events/login.adoc:162]:
```suggestion
The Email Event Listener sends a message to the user's email address when an event occurs and supports the following events:
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Approved with one change.

**Comment by ahus1** [comment]
@mposolda - please re-approve when you have the time. Thanks!

**Review by andymunro — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**